runtime/development/src/lib.rs

```diff
@@ -936,6 +936,56 @@ impl PoolUpdateGuard for UpdateGuard {
 	}
 }
 
+pub struct CurrencyPriceSource;
+impl CurrencyPrice<CurrencyId> for CurrencyPriceSource {
+	type Rate = Rate;
+	type Moment = Moment;
+
+	fn get_latest(
+		base: CurrencyId,
+		quote: Option<CurrencyId>,
+	) -> Option<PriceValue<CurrencyId, Self::Rate, Self::Moment>> {
+		match base {
+			CurrencyId::Tranche(pool_id, tranche_id) => {
+				let now = <pallet_timestamp::Pallet<Runtime> as UnixTime>::now().as_secs();
```

Comment:
This should probably return `None` if the requested `quote` parameter isn't `None` or the pool's currency?

Reply:
Misread the above, will add a check that `quote` parameter isn't `None` or the pool's currency

Reply:
Fixed now.